packages/webdriver/package.json

```diff
@@ -36,5 +36,8 @@
     "@wdio/utils": "5.18.6",
     "lodash.merge": "^4.6.1",
     "request": "^2.83.0"
+  },
+  "devDependencies": {
+    "@types/request": "^2.48.4"
```

Comment:
This needs be a `dependency` because it's now used in the types this package exports.